sprints/sustainability/tasks.py

```diff
@@ -56,3 +67,71 @@ def validate_worklog_cache(long_term=True, force_regenerate=False) -> bool:
 
     cache.delete(settings.CACHE_WORKLOG_REGENERATE_LOCK)
     return True
+
+
+@celery_app.task()
+def send_email_alerts() -> bool:
+    """
+    Send email alerts to users that keep track of the sustainability and budgets.
+
+    Email addresses can be specified via:
+        - `NOTIFICATIONS_SUSTAINABILITY_EMAILS` env variable - this subscribes an email for ALL notifications,
+        - `Cell`/`Account` models of Django admin - only for specific cells/accounts.
+
+    Note: notifications for projects that are not added to `Cells` (e.g. via Django admin) will be skipped for brevity.
+    """
+    today = datetime.today()
+    last_day_of_month = calendar.monthrange(today.year, today.month)[1]
+    start_date = today.replace(day=1)
+    end_date = today.replace(day=last_day_of_month)
+    start_str = start_date.strftime(settings.JIRA_API_DATE_FORMAT)
+    end_str = end_date.strftime(settings.JIRA_API_DATE_FORMAT)
+    dashboard = SustainabilityDashboard(start_str, end_str)
+
+    # Send alerts for cells.
+    max_percentage_ratio = settings.SUSTAINABILITY_MAX_NON_BILLABLE_TO_BILLABLE_CELL_RATIO * 100
+    _send_email_alert(
+        dashboard.get_projects_sustainability(),
+        lambda k, v: v >= max_percentage_ratio and Cell.objects.filter(name=k).exists(),  # Ignore non-set projects.
```

Comment:
Instead of this cryptic line, what do you think of filtering the items beforehand and just passing to `_send_email_alert` whatever needs to be emailed?

This would make this behavior more explicit, easier to understand and remove the filtering from the function that send emails.

Reply:
Good idea to separate this. Done.